src/sxastarter/package.json

```
@@ -29,15 +29,16 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@sitecore-jss/sitecore-jss-nextjs": "^20.1.0-canary",
+    "@sitecore-jss/sitecore-jss-nextjs": "^21.0.0-canary",
     "bootstrap": "^5.1.3",
     "font-awesome": "^4.7.0",
     "graphql": "~15.8.0",
     "graphql-tag": "^2.11.0",
-    "next": "^12.2.0",
-    "next-localization": "^0.10.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "next": "^12.2.5",
+    "next-localization": "^0.12.0",
+    "react": "^18.1.0",
+    "react-dom": "^18.1.0",
+    "@sitecore/engage": "^0.3.0"
```

0.3.1 was just release today and we will be incorporating in JSS 21 (see [pending PR](https://github.com/Sitecore/jss/pull/1183/files#diff-c0ec57ebe5df7d0c241aa40a862a09b363930f5a79bdd8e43d47c942c99b041d)).
```suggestion
    "@sitecore/engage": "^0.3.1"
```